### ISSUE TITLE
[tfjs-tflite] ERROR: Didn't find op for builtin opcode 'CONV_2D' version '6'.

### ISSUE THREAD

**Issue by josephrocca** [tensorflow/tfjs#6885]
I think the TFLite runtime may need to be updated/recompiled? I'm trying to get the new version of [Google's Lyra](https://opensource.googleblog.com/2022/09/lyra-v2-a-better-faster-and-more-versatile-speech-codec.html) (open-sourced a few days ago) working on the web.

Tested on Chrome 105. You can see tfjs and tfjs-tflite versions in the URLs below.

## Minimal reproduction:
```html
<script src="https://cdn.jsdelivr.net/npm/@tensorflow/tfjs-core@3.20.0"></script>
<script src="https://cdn.jsdelivr.net/npm/@tensorflow/tfjs-backend-cpu@3.20.0"></script>
<script src="https://cdn.jsdelivr.net/npm/@tensorflow/tfjs-tflite@0.0.1-alpha.8/dist/tf-tflite.min.js"></script>
  
<script>
  tflite.loadTFLiteModel("https://huggingface.co/rocca/lyra-v2-soundstream/resolve/main/tflite/soundstream_encoder.tflite");
</script>
```

## JSBin Example:
https://jsbin.com/cunavuxewe/edit?html,output

## Full error:
```
ERROR: Didn't find op for builtin opcode 'CONV_2D' version '6'. An older version of this builtin might be supported. Are you using an old TFLite binary with a newer model?
```

![image](https://user-images.githubusercontent.com/1167575/193438422-fa04d8ea-28e3-4eed-9895-d983268433df.png)

**Comment by jinjingforever** [comment]
Hi @josephrocca 

Thanks for the report! I just released tfjs-tflite 0.0.1-alpha.9 with the latest tflite binary which seems to have fixed the problem (I tried it with your jsbin example). Please give it a try.

Thanks!

**Comment by josephrocca** [comment]
Thank you so much for the quick resolution!!
